From 46c38875897ac19c998d350cc4486c16e173c57b Mon Sep 17 00:00:00 2001
From: "Teoh, Ji Sheng" <ji.sheng.teoh@intel.com>
Date: Sat, 7 May 2022 09:22:04 +0000
Subject: [PATCH] arm: stratix10: add board configuration

This patch adds the following configuration:
board-4: Combined GHRD design

Upstream-Status: Pending

Signed-off-by: Teoh, Ji Sheng <ji.sheng.teoh@intel.com>
---
 arch/arm/dts/socfpga_stratix10_socdk-u-boot.dtsi | 12 ++++++++++++
 1 file changed, 12 insertions(+)

diff --git a/arch/arm/dts/socfpga_stratix10_socdk-u-boot.dtsi b/arch/arm/dts/socfpga_stratix10_socdk-u-boot.dtsi
index 24d47fcd12..30292eb506 100755
--- a/arch/arm/dts/socfpga_stratix10_socdk-u-boot.dtsi
+++ b/arch/arm/dts/socfpga_stratix10_socdk-u-boot.dtsi
@@ -89,6 +89,18 @@
 			sign-images = "atf", "uboot", "fdt-1";
 		};
 	};
+
+	board-4 {
+		description = "board_4";
+		firmware = "atf";
+		loadables = "uboot";
+		fdt = "fdt-0";
+		signature {
+			algo = "crc32";
+			key-name-hint = "dev";
+			sign-images = "atf", "uboot", "fdt-0";
+		};
+	};
 };
 
 &binman {
-- 
2.17.1

